app/code/Magento/Catalog/Test/Unit/Controller/Category/ViewTest.php

```diff
@@ -30,219 +36,239 @@
 use Magento\Store\Model\StoreManagerInterface;
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
 
 /**
  * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  * @SuppressWarnings(PHPMD.TooManyFields)
  */
 class ViewTest extends TestCase
```

Comment:
I think it's better to remove this unit test in favour to MFTF tests that are more suitable for controller testing.

As I can see the Unit test is covering the category rendering in different Display Modes, @Bartlomiejsz do you think you can replace the Unit with MFTF test? I believe that would be a valuable improvement

Reply:
Hi @sivaschenko, regarding this - maybe you're right that mftf would fit here better than unit test, but this one is almost completely not related to my changes.
I had to modify it, because I added retrieving of path info from request, but what is tested there is not related.
Also, I fear I won't have chance in short future to create additional mftf tests based on this existing.

I covered my change with mftf, so I'd suggest to not block this PR because of such part, that could of course be improved.